.github/workflows/ci.yml

```diff
@@ -159,3 +171,8 @@ jobs:
     - if: "!startsWith(matrix.os, 'windows') && !contains(matrix.target, 'ios')"
       run: cargo test
       name: Run tests (not Windows)
+    - if: "contains(matrix.target, 'ios')"
+      # It's... theoretically possible to run tests on iPhone Simulator,
+      # but for now, make sure that BoringSSL only builds.
+      run: cargo build --target ${{ matrix.target }}
```

Comment:
This is `build` instead of check to make sure the linker succeeds?

Reply:
Hm... I think `check` would work fine here too, as there is no linker step. Rust does not really support producing cdylibs for iOS targets, only staticlibs, and there is no linkage there. `boring` crate itself does not provide `staticlib`, so this only builds an `rlib`, which is not really useful for building iOS targets.

Reply:
You know what, I have a better idea. How about building the tests, just `--no-run` them? Doing `cargo check` will omit all `#[cfg(test)]` code from compilation – which should be still checked by Clippy, I think, but nevertheless. Maybe some time later someone thinks up a way how to run the binaries found in the target directory.

Reply:
If you want to check tests too you can use `cargo check --all-targets`.

Reply:
`cargo test --no-run` with `cargo check --all-targets` then.